include/boost/histogram/python/register_histogram.hpp

```diff
@@ -246,9 +246,10 @@ py::class_<bh::histogram<A, S>> register_histogram(py::module &m, const char *na
 
                 namespace bmp = boost::mp11;
                 static_assert(
-                    bmp::mp_empty<bmp::mp_set_difference<
-                        bmp::mp_unique<bmp::mp_transform<::detail::get_axis_value_type, bmp::mp_first<axes::any>>>,
-                        bmp::mp_list<double, int, std::string>>>::value,
+                    bmp::mp_empty<
+                        bmp::mp_set_difference<bmp::mp_unique<bmp::mp_transform<::detail::get_axis_value_type,
+                                                                                bmp::mp_first<axis_variant_vector>>>,
```

Comment:
I think you could drop the first here if you could directly access `axis_variant`

Reply:
Ok, having `axis_variant` separate also makes sense.